packages/typedoc-plugin-appium/lib/output/theme/utils.ts

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import fs from 'node:fs';
+import path from 'node:path';
+import Handlebars from 'handlebars';
+import {ContainerReflection, PageEvent, ReflectionKind} from 'typedoc';
+import {AppiumPluginReflectionKind} from '../../model';
+
+const RESOURCES_PATH = path.join(__dirname, '..', '..', '..', 'resources');
```

Comment:
I's rather use path.abosulte

Reply:
what do you mean?

Reply:
isn't `__dirname` guaranteed to be absolute?

Reply:
I mean that the resulting string would look like "/bla/../../../resources" if you use `join` and something like `/resources` if absolute path is used. My point is that it is much easier to parse absolute paths in logs in comparison to relative

Reply:
I still don't understand.  Do you mean `path.resolve()`?  Or `path.relative()`?  What does this have to do with logging?

Reply:
yes, path.resolve should do the job

Reply:
well, what @jlipps was saying, it doesn't need to be `path.resolve()` since `__dirname` is always absolute, so `path.join()` will be the same thing.

Reply:
I now understand why @mykola-mokhnach prefers path.resolve. It will eliminate `../../..` in the resulting path, which makes looking at the path as a human more sensible (because you don't have to go down three directories and back up). we might look at this path as a human if e.g. it is printed out in logs.